packages/lesswrong/components/common/ContentStyles.tsx

```diff
@@ -15,7 +15,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     '& h1, & h2, & h3': {
       fontSize: isFriendlyUI ? "1.1rem" : "1.6rem",
       // Cancel out a negative margin which would cause clipping
-      marginBlickStart: "0 !important",
+      marginBlockStart: "0 !important",
```

Comment:
This is fixing an issue that's been around for years, characterized by e.g. posts in recent discussion that start with an h1 header having that header clipped at the top (since it has negative margin by default).  Not forum-gating this, but giving @Will-Howard @oetherington @s-cheng a heads-up